UNRELEASED.md

```diff
@@ -0,0 +1,8 @@
+**Breaking:**
+
+- bpk-component-accordion
+  - Remove `weight` property from `BpkAccordion`.
```

Comment:
This should be removed from here as the latest in `main` does not have this due to the recent Backpack release 😄